src/EnergyPlus/EconomicLifeCycleCost.hh

```diff
@@ -501,7 +501,7 @@ struct EconomicLifeCycleCostData : BaseGlobalStruct
 
     // present value factors
     Array1D<Real64> SPV;
-    std::map<int, std::map<DataGlobalConstants::ResourceType, Real64>> energySPV; // yearly equivalent to FEMP UPV* values
+    std::map<int, std::array<Real64, static_cast<int>(Constant::ResourceType::Num)>> energySPV; // yearly equivalent to FEMP UPV* values
```

Comment:
Here is the data structure change, the outer map (year-indexed) is still a `std::map`.